### DISCUSSION TITLE
Question: Is Xamarin Froms 5.x or 6 going to Support .Net 6

### DISCUSSION THREAD

**Discussion by dgerding** [xamarin/Xamarin.Forms#14845] [General]
Hi All,
With Maui delayed, I know Xamarin Forms is in more of a maintenance mode. Will there be a version that targets .net 6? If so, when?

Thanks,
Dave Gerding

**Comment by KarlSearl** [comment]
Hi,  

From my limited understanding,  MAUI is the answer to your question.

**Comment by jfversluis** [comment]
There will be no .NET 6 version of Xamarin.Forms for more on the background read here #14829
